rpm/centos-7/Dockerfile

```diff
@@ -11,7 +11,7 @@ ENV GOPATH=/go
 ENV PATH $PATH:/usr/local/go/bin:$GOPATH/bin
 ENV AUTO_GOPATH 1
 ENV DOCKER_BUILDTAGS pkcs11 seccomp selinux
-ENV RUNC_BUILDTAGS seccomp selinux
+ENV RUNC_BUILDTAGS seccomp selinux nokmem
```

Comment:
The `runc` version that ships with the centos7 packages is in the containerd.io package; actually wondering if this `ENV` is used at all (if it is, it would only be used for the static builds, which are not distro-specific: https://download.docker.com/linux/static/stable/x86_64/)

Reply:
ping @seemethere

Reply:
Is the packaging for the containerd.io packages that are provided by Docker available anywhere?

Reply:
The repository containing the packaging scripts is not yet public; work is being done on open-sourcing those scripts, but there's still some refactoring to be done (there's some hard-coded parts in there that are specific to Docker's internal infrastructure etc.).

I think the plan is to donate it to the containerd project, and have it live somewhere in the https://github.com/containerd org at some point.